src/components/bcros/Alert.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <div data-cy="alert-display" class="px-3 py-3">
+    <h3>
+      <UIcon
+        :class="`${iconColour} mr-2`"
+        :name="iconName"
+        data-cy="contact-icon"
+      />
+      <span class="font-semibold flex-auto">{{$t(alertHeader)}}</span>
+      <button class="float-right text-blue-600 hover:bg-blue-100 text-sm px-1 py-1mr-4" href="#" @click="expanded = !expanded">
+        {{ !expanded ? 'View Details' : 'Hide Details' }}
+        <UIcon
+          :class="`text-blue-600 mr-2`"
+          :name="!expanded ? 'i-mdi-chevron-down' : 'i-mdi-chevron-up'"
+          data-cy="contact-icon"
+        />
+      </button>
+    </h3>
+    <div v-if="expanded">
```

Comment:
Also is there a reason you didn't use the UAccordian for the alerts? That might simplify things in here quite a bit

Reply:
discussed in chat to keep it as is for now and update later to use code that is still in progress